include/vrv/annotscore.h

```diff
@@ -0,0 +1,97 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        annotscore.h
+// Author:      David Lewis
+// Created:     2024
+// Copyright (c) Authors and others. All rights reserved.
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef __VRV_ANNOTSCORE_H__
+#define __VRV_ANNOTSCORE_H__
+
+#include "atts_cmn.h"
+#include "atts_shared.h"
+#include "controlelement.h"
+#include "editorial.h"
+#include "plistinterface.h"
+#include "timeinterface.h"
+
+namespace vrv {
+
+//----------------------------------------------------------------------------
+// AnnotScore
+//----------------------------------------------------------------------------
+
+/**
+ * This class models the MEI <annot> element where @type is score.
+ */
+class AnnotScore : public ControlElement, public PlistInterface, public TimeSpanningInterface {
+public:
+    /**
+     * @name Constructors, destructors, and other standard methods
+     * Reset method reset all attribute classes
+     */
+    ///@{
+    AnnotScore();
+    virtual ~AnnotScore();
+    // Object *Clone() const override { return new AnnotScore(*this); }
+    void Reset() override;
+    std::string GetClassName() const override { return "annotScore"; }
```

Comment:
Element name should still be `annot`
```suggestion
    std::string GetClassName() const override { return "annot"; }
```